### ISSUE TITLE
Medspacy failing to install

### ISSUE THREAD

**Issue by ssamine** [medspacy/medspacy#130]
installing medspacy always results to this:
```
`Collecting spacy<3.2.0,>=3.1.3
  Using cached spacy-3.1.6-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (6.0 MB)
Collecting PyRuSH>=1.0.3.5
  Using cached PyRuSH-1.0.3.5.tar.gz (44 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [63 lines of output]
      /opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/dist.py:767: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead
        warnings.warn(
      /opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
        error: subprocess-exited-with-error
      
        × python setup.py egg_info did not run successfully.
        │ exit code: 1
        ╰─> [13 lines of output]
            Traceback (most recent call last):
              File "<string>", line 2, in <module>
              File "<pip-setuptools-caller>", line 34, in <module>
              File "/tmp/pip-wheel-9y2qbowd/quicksectx_044b4b02408042e08e3b3f3691ce81cd/setup.py", line 50, in <module>
                ext_modules=cythonize(extensions, language_level=3, language='c++'),
              File "/opt/conda/envs/torch113/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 970, in cythonize
                module_list, module_metadata = create_extension_list(
              File "/opt/conda/envs/torch113/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 816, in create_extension_list
                for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
              File "/opt/conda/envs/torch113/lib/python3.9/site-packages/Cython/Build/Dependencies.py", line 114, in nonempty
                raise ValueError(error_msg)
            ValueError: 'quicksect/quicksect.pyx' doesn't match any files
            Warning: passing language='c++' to cythonize() is deprecated. Instead, put "# distutils: language=c++" in your .pyx or .pxd file(s)
            [end of output]
      
        note: This error originates from a subprocess, and is likely not a problem with pip.
      error: metadata-generation-failed
      
      × Encountered error while generating package metadata.
      ╰─> See above for output.
      
      note: This is an issue with the package mentioned above, not pip.
      hint: See above for details.
      Traceback (most recent call last):
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/installer.py", line 82, in fetch_build_egg
          subprocess.check_call(cmd)
        File "/opt/conda/envs/torch113/lib/python3.9/subprocess.py", line 373, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['/opt/conda/envs/torch113/bin/python', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpcd35vgj3', '--quiet', 'quicksectx']' returned non-zero exit status 1.
      
      The above exception was the direct cause of the following exception:
      
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-tqcq38r6/pyrush_5672d33671b7425fba259511bb868f97/setup.py", line 36, in <module>
          setup(
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/__init__.py", line 86, in setup
          _install_setup_requires(attrs)
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/__init__.py", line 80, in _install_setup_requires
          dist.fetch_build_eggs(dist.setup_requires)
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/dist.py", line 871, in fetch_build_eggs
          resolved_dists = pkg_resources.working_set.resolve(
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/pkg_resources/__init__.py", line 789, in resolve
          dist = best[req.key] = env.best_match(
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/pkg_resources/__init__.py", line 1075, in best_match
          return self.obtain(req, installer)
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/pkg_resources/__init__.py", line 1087, in obtain
          return installer(requirement)
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/dist.py", line 951, in fetch_build_egg
          return fetch_build_egg(self, req)
        File "/opt/conda/envs/torch113/lib/python3.9/site-packages/setuptools/installer.py", line 84, in fetch_build_egg
          raise DistutilsError(str(e)) from e
      distutils.errors.DistutilsError: Command '['/opt/conda/envs/torch113/bin/python', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpcd35vgj3', '--quiet', 'quicksectx']' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Note: you may need to restart the kernel to use updated packages.
```

**Comment by ck37** [comment]
Please see #128

**Comment by burgersmoke** [comment]
I believe that this issue was due to Python 3.9 not being supported.  In medspacy releases before 1.0.0 (e.g., 0.2.0.0), the solution was to use Python 3.8.

With medspacy 1.0.0 now released, Python 3.9 is now supported as well.  If there are further issues, I propose that new issues be opened for this 1.0.0 release.
